pybikes/gobike.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2015, Ben Caller <bcaller@gmail.com>
+# Distributed under the AGPL license, see LICENSE.txt
+
+import json
+
+from lxml import html
+from lxml.cssselect import CSSSelector
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+from contrib import TSTCache
+
+__all__ = ['GoBike', 'GoBikeStation']
+
+PAGE_SIZE = 10
+STATION_LIST_PATH = '/umbraco/Surface/BookingSurface/GetStationsList'
+
+cache = TSTCache(delta=60)
```

Comment:
The TSTCache is meant to be used on cases where two systems point to the same URL (for example, nextbike). Is this the case for this system?